scripts/pylib/pytest-twister-harness/src/twister_harness/plugin.py

```diff
@@ -36,8 +36,13 @@ def pytest_addoption(parser: pytest.Parser):
         type=float,
         default=60.0,
         help='Set base timeout (in seconds) used during monitoring if some '
-             'operations are finished in a finite amount of time (e.g. waiting '
-             'for flashing).'
+             'operations are finished in a finite amount of time.'
+    )
+    twister_harness_group.addoption(
+        '--flash-timeout',
```

Comment:
could you please also add support for `--device-flash-with-test` at pytest plugin ?

Reply:
I do not really see any use case for this in pytest-harness. Flashing is done separately (in pytest fixture), test case is a separate part.